test/protractor.conf.js

```diff
@@ -19,4 +20,5 @@ module.exports.config = {
     includeStackTrace: true,
     random: true
   }
+  // highlightDelay: 1000
```

Comment:
We're using 'headless', this isn't useful for us, Why keep?

Reply:
When debugging I comment the headless config and this turns to be useful.